src/azure-cli/azure/cli/command_modules/signalr/tests/latest/test_signalr_commands.py

```diff
@@ -89,14 +96,15 @@ def test_signalr_commands(self, resource_group):
         ])
 
         # Test update
-        self.cmd('az signalr update -n {signalr_name} -g {rg} --sku {updated_sku} --tags {updated_tags} --service-mode {update_service_mode}',
+        self.cmd('az signalr update -n {signalr_name} -g {rg} --sku {updated_sku} --tags {updated_tags} --service-mode {update_service_mode} --enable-message-logs {update_enable_message_logs}',
```

Comment:
please add test for `--enable-message-logs false` in not string variable. 
Check value with boolean False, not string if the output type is boolean for features[1].value

Reply:
@Juliehzl  Test added for `--enable-message-logs false`.
The check value is string since the [SignalrFeature](https://docs.microsoft.com/en-us/rest/api/signalr/signalr/createorupdate#signalrfeature).value is a string.